project/Dependencies.scala

```diff
@@ -19,7 +19,12 @@ import sbt.*
 
 object Dependencies {
 
-  lazy val common = core ++ infoset ++ test
+  def common(scalaVersion: String) = core ++ infoset ++ test ++ {
+    CrossVersion.partialVersion(scalaVersion) match {
+      case Some((2, 13)) => Seq("org.scala-lang" % "scala-reflect" % "2.13.16")
```

Comment:
It looks like before this change scala-reflect was only needed by the macroLib. Should this be removed from common and instead this logic added to build.sbt with the daffodil-macro project?